index.js

```diff
@@ -510,8 +510,12 @@ function decorateNextFn(fn) {
     called = true;
     // Make sure to clear the stack so try/catch doesn't catch errors
     // in subsequent middleware
-    return process.nextTick(() => fn.apply(_this, arguments));
+    return nextTick(() => fn.apply(_this, arguments));
   };
 }
 
+const nextTick = process && process.nextTick || function (cb) { 
```

Comment:
Maybe `typeof process !== 'undefined' && process.nextTick`? That way, doesn't crash in environments where `process` isn't defined

Also, `function nextTick(cb) {` because that means `nextTick` will show up in stack traces rather than anonymous function

Reply:
```suggestion
const nextTick = typeof process === 'object' && process !== null && process.nextTick || function nextTick(cb) { 
```

How about that?

Reply:
Btw good point regarding the stacktrace.